src/util.cc

```diff
@@ -368,6 +368,15 @@ std::string ToLower(std::string in) {
   return in;
 }
 
+bool CompareString(const std::string& lhs, const std::string& rhs, bool case_insensitive) {
```

Comment:
I think we just care about case-insensitive situation is fine, we do not need the function while case-sensitive, and it has a worse performance since it has a runtime `case_insensitive` checking.